test-framework/clustering/src/main/java/org/keycloak/testframework/server/ClusteredKeycloakServer.java

```diff
@@ -94,12 +111,16 @@ private void startContainersWithSameImage(KeycloakServerConfigBuilder configBuil
             containers[i] = container;
 
             copyProvidersAndConfigs(container, configBuilder);
-
-            container.setCustomLogConsumer(new JBossLogConsumer(Logger.getLogger("managed.keycloak." + i)));
+            configureLogConsumers(container, i, clusterLatch);
             container.run(configBuilder.toArgs());
         }
     }
 
+    private void configureLogConsumers(DockerKeycloakDistribution container, int index, CountdownLatchLoggingConsumer clusterLatch) {
+        container.setCustomLogConsumer(new JBossLogConsumer(Logger.getLogger("managed.keycloak." + index)));
+        container.setCustomLogConsumer(clusterLatch);
```

Comment:
Setting the `clusterLatch` consumer removes the  `JBossLogConsumer`.
```suggestion
        var logger = new JBossLogConsumer(Logger.getLogger("managed.keycloak." + index));
        container.setCustomLogConsumer(logger.andThen(clusterLatch));
```